client/src/components/styles/Profile.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  background-color: #f4f4f4;
+}
+
+.profile {
+  margin: 0 auto;
+}
+
+.profile-name {
+  font-weight: 700;
+  font-size: 1.3rem;
+}
+
+.button {
+  font-size: 0.9rem;
+}
+
+.profile-info {
+  flex-direction: column;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+  font-size: 1rem;
+}
+
+a:hover {
+  text-decoration: none;
+  color: black;
+}
```

Comment:
that's not scss style
where is nesting?

For example
```
a {
  text-decoration: none;
  color: #000;
  font-size: 1rem;
  
  &:hover {
      text-decoration: none;
      color: black;
  }
}
```

Reply:
Fixed